packages/chakra-ui/package.json

```
@@ -24,11 +24,13 @@
     "validate": "npm run lint && npm run prettier -- --list-different && npm run tslint",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "npm install && build-storybook",
-    "build:watch": "nodemon -w src -x 'rm .babelrc &> /dev/null; npm run build'"
+    "build:watch": "nodemon -w src -x 'rm .babelrc &> /dev/null; npm run build'",
+    "test": "jest --watch"
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged"
+      "pre-commit": "npm test && lint-staged",
+      "pre-push": "npm test && lint-staged"
```

As the number of test cases grows, having to wait for tests to finish before commit *and* before push will become very cumbersome. I think we should decide on one or the other.

---

```suggestion
      "pre-commit": "lint-staged",
      "pre-push": "npm run test"
```

I think it should be like this

---

The only reason I had it running both times is because it's annoying to make a commit, and then push, only to realize that there are failing tests, requiring either another commit or to reset the previous one.

I'm willing to do the `pre-push` route and treat it as more of a reminder mechanism than anything else.